chore: Create how-to folder for upgrades (#435)
CluedIn-io/CluedIn.Documentation: docs/190-paas-operations/002-upgrade.md

has_children: true
tags: ["deployment", "kubernetes", "azure", "aks", "microsoft", "marketplace", "azure-marketplace"]
headerIcon: "paas"
---

When signing up with CluedIn and taking on the Managed Services agreement, Product upgrades will be handled for you by the CluedIn team. The team will reach out and work with your requirements to ensure that upgrades happen when it's most convenient for your business.

> For some customers, a more tailored agreement exists whereby the management of the cluster and resources are managed by you, including upgrades. Below is documentation on how to self-service upgrading of CluedIn.

Reviewer: ```suggestion
For some customers, a more tailored agreement exists whereby the management of the cluster and resources are managed by you, including upgrades. This section contains guides for self-service upgrading of CluedIn.
```

> ## Release Notes
> For release note specifics, please visit [Release Notes](/release-notes)

Reviewer: ```suggestion
{:.important}
For release note specifics, please visit [Release Notes](/release-notes).
```
